src/packet/bwe/mod.rs

```diff
@@ -231,3 +240,149 @@ impl fmt::Display for BandwidthUsage {
         }
     }
 }
+
+/// Sliding window [`SeqNo`]s storage.
+///
+/// Only remembers the last [`SIZE`] packets added.
+#[derive(Debug)]
+struct HandledPacketsTracker<const SIZE: usize> {
+    /// Recently added packets.
+    history: [Option<u16>; SIZE],
```

Comment:
Optimization: Do we really need to remember the value here? If we use a bit vector we can get away with 32 bytes for 256 entries instead of 4096. We could use the savings to increase the max size to 512 entries.

Reply:
Reworked to use bit vec in [65c5446](https://github.com/algesten/str0m/pull/601/commits/65c544620c40aa4e2d534feb80ea6afaebec1180). It has some downsides under heavy reordering scenarios, but should to the job fine in the real world.